include/z64audio.h

```diff
@@ -59,6 +59,19 @@ typedef enum {
     /* 3 */ CACHE_PERMANENT
 } AudioCacheType;
 
+typedef enum {
+    /* 0 */ LOAD_STATUS_NOT_LOADED,
+    /* 1 */ LOAD_STATUS_IN_PROGRESS,
+    /* 2 */ LOAD_STATUS_COMPLETE,
+    /* 3 */ LOAD_STATUS_DISCARDABLE,
+    /* 4 */ LOAD_STATUS_MAYBE_DISCARDABLE,
+    /* 5 */ LOAD_STATUS_PERMANENTLY_LOADED
+} AudioLoadStatus;
+
+#define IS_SAMPLE_FONT_LOAD_COMPLETE(sampleBankId) (gAudioContext.sampleFontLoadStatus[sampleBankId] >= LOAD_STATUS_COMPLETE)
+#define IS_FONT_LOAD_COMPLETE(fontId) (gAudioContext.fontLoadStatus[fontId] >= LOAD_STATUS_COMPLETE)
+#define IS_SEQ_LOAD_COMPLETE(seqId) (gAudioContext.seqLoadStatus[seqId] >= LOAD_STATUS_COMPLETE)
+
```

Comment:
I think I would almost prefer if the word "is" was taken out of these macro names
I know we have things like `IS_NIGHT` and `IS_DAY` on the game side of things, but overall it feels weird to have a macro named like that

Probably an unpopular opinion, can see how others feel. but for example i think 
`if (SAMPLE_FONT_LOAD_COMPLETE(x)`
seems fine

Reply:
I like the `is` prefix better but it's ok without it